webpack.dev.config.js

```diff
@@ -1,5 +1,4 @@
 var common = require('./webpack.common.config');
 var devConfig = common('okta-sign-in.js');
-devConfig.devtool = '#inline-source-map';
```

Comment:
can we keep the dev build a little faster? Maybe explore other options for [devtool](https://webpack.js.org/configuration/devtool/)

I also heard @richardheng-okta researched this values recently...

Reply:
https://webpack.js.org/configuration/devtool/ `inline-source-map` is the same as `source-map` in building time, the change affects `grunt webpack:release` only.

Reply:
no, the change to ```webpack.common.config.js``` affects all builds, release, dev and test. (although dev was using ```inline-source-map``` as you pointed out, so the build time is basically the same)... but that is why I was suggesting maybe explore other option for dev... 
I would be ok if we want to do that later as well.

Reply:
well, I mean, with my change, the build time doesn't change a lot for *dev environment*, because `inline-source-map` is same as `source-map` in building performance. I can't foresee a space to optimize if we want to have a version that allows us to debug. The fact is unchanged with my change: dev would be faster (same faster space with my change) than release for 2 reasons: dev version is not uglified, thus the source-map is faster to build; and dev only need to build one js file, while release need to build 3.

Reply:
If you decide to change it in the future, I have found that Chrome handles inline-source-map a bit better during debugging: https://github.com/webpack/webpack/issues/2145#issuecomment-325441145

Reply:
okay, I reverted to use `inline-source-map`, thanks for the link, it provides more info than the [documentation of devtool](https://webpack.js.org/configuration/devtool/).